modules/040-control-plane-manager/docs/FAQ_RU.md

```diff
@@ -4,7 +4,58 @@ title: "Управление control plane: FAQ"
 
 ## Как добавить master-узел?
 
-Поставить на узле кластер label `node-role.kubernetes.io/master: ""`, все остальное произойдет полностью автоматически.
+### Cтатичный или гибридный кластер
+Чтобы добавить статичный master-узел в кластер, выполните следующие действия:
+- [Добавьте](https://deckhouse.io/ru/documentation/v1/modules/040-node-manager/faq.html#как-автоматически-добавить-статичный-узел-в-кластер) в кластер статичный узел. `NodeGoup` может быть любая, с параметром `NodeType` — `Static`.
```

Comment:
Oh, no. To add master nodes to a static cluster, you need to use the manual-boostrap-for-master script, as per other nodes of other node groups. Thus, here you only need to write something like 
> Для добавления мастер ноды в статический кластер, воспользуйтесь инструкцией[ссылка] для добавления статической ноды. Компоненты control-plane будут добавлены на ноду автоматически.

Reply:
Done